imod/mf6/simulation.py

```diff
@@ -273,9 +280,9 @@ def write(
                     value.write(key, globaltimes, ims_write_context)
             elif isinstance(value, list):
                 for exchange in value:
-                    if isinstance(exchange, imod.mf6.GWFGWF):
+                    if isinstance(exchange, imod.mf6.exchangebase.ExchangeBase):
                         exchange.write(
-                            exchange.packagename(), globaltimes, write_context
+                            exchange.package_name(), globaltimes, write_context
```

Comment:
globaltimes -> global_times?

Reply:
``globaltimes`` is consistent with the rest. The name ``globaltimes`` is used throughout the ``mf6`` module, renaming it here is out of scope.